src/dataset.py

```diff
@@ -9,24 +9,33 @@
 import torchvision.transforms as T
 
 import src.utils as utils
+import src.augmentations as TA
 torch.manual_seed(10)
 
 class GlacierDataset(Dataset):
     def __init__(self, base_dir, data_file, channels_to_inc=None, img_transform=None,
                  mode='train', borders=False, use_cropped=True, use_snow_i=False,
-                 mask_used='glacier'):
+                 use_elev=True, use_slope=True, mask_used='glacier', country='all', year='all'):
         super().__init__()
         self.base_dir = base_dir
         data_path = Path(base_dir, data_file)
         self.data = pd.read_csv(data_path)
         self.data = self.data[self.data.train == mode]
         if mask_used == 'debris_glaciers':
             self.data = self.data[self.data.pseudo_debris_perc > 0]
+        if country != 'all':
+            self.data = self.data[self.data["country"].isin(country)]
+        if year != 'all':
+            self.data = self.data[self.data["year"].isin(year)]
         self.img_transform = img_transform
         self.borders = borders
         self.use_cropped = use_cropped
         self.use_snow_i = use_snow_i
-        self.channels_to_inc = channels_to_inc
+
+        if channels_to_inc is not None: self.channels_to_inc = channels_to_inc[:]
```

Comment:
I feel like it's more conventional to indent here, even though I get that this is more concise.